manifest.json

```diff
@@ -41,6 +41,15 @@
         "description": "Enter the ID (GTM-XXXX) from your Google Tag Manager",
         "type": "string"
       },
+      "typeScript": {
+        "title": "Script attribute type",
+        "description": "Define whether the type attribute execute script on a secondary thread or not.",
+        "default": "javascript",
+        "enum": [
+          "javascript",
+          "partytown"
+        ]
+      },
```

Comment:
What do you think about turning this setting into a boolean like `usePartytown`? Since there isn't a third option, it's safer and simpler.

Reply:
@danzanzini Yes. I don't think there will be a third option anyway.